example/fasthttp/main.go

```diff
@@ -0,0 +1,67 @@
+package main
+
+import (
+	"fmt"
+
```

Comment:
nit: This empty line is some Go hypster thing?

Reply:
`gofmt` itself adds new line between native and external imports.